features/pages/location_directory_page.rb

```diff
@@ -34,6 +35,20 @@ def hide_active_locations
     filter_submit.click unless javascript_enabled?
   end
 
+  def hide_location(_location)
```

Comment:
I guess this was to satisfy the rubocop gods? I'm not sure if this is the default rule for unused arguments in a method signature, but I tend to prefer `*` for variadic/unused arguments. But this is up for discussion as it is probably quite personal.